packages/flutter/test/material/checkbox_list_tile_test.dart

```diff
@@ -50,11 +50,11 @@ void main() {
 
     await tester.pumpWidget(buildFrame(null));
     await tester.pumpAndSettle();
-    expect(getCheckboxListTileRenderer(), paints..path(color: const Color(0xFFFFFFFF)));
+    expect(getCheckboxListTileRenderer(), paints..path(color: const Color(0xff1e88e5))..path(color: const Color(0xFFFFFFFF)));
```

Comment:
This kind of thing is typically formatted (elsewhere in this file) as 
```dart
expect(getCheckboxListTileRenderer(), 
  paints
    ..path(color: const Color(0xff1e88e5))
    ..path(color: const Color(0xFFFFFFFF))
 );
```

It would help if the colors were bound to vars with names that indicated what part of the checkbox the color was being applied to.